samples/quickstart/model.yaml

```diff
@@ -1,11 +1,19 @@
+apiVersion: v1
```

Comment:
this is a great example. However, for quick start demos, we hope the experiences is minimum and mainly used for testing purpose. Do you mind add a separate page talking about the model deployment practice instead?